src/parser/mage/fire/modules/talents/MeteorCombustion.tsx

```diff
@@ -41,10 +44,18 @@ class MeteorCombustion extends Analyzer {
   }
 
   onCombustionStart(event: ApplyBuffEvent) {
-    this.combustionActive = true;
+    //The Sun King's Blessing Legendary effect has a chance to trigger a 6sec Combust which was throwing this stat off, so we are just checking to see if Combustion was cast within 100ms of the buff being applied.
```

Comment:
So there are a few reasons i did this.

1. The statistic was showing stuff like 200% efficiency because it was counting the applybuff instead of cast (which is primarily to catch combustions that are cast just barely before the fight starts)
2. Maybe you can cast Meteor in the 6sec Combust, but i dont think it should count against you if you dont
3. If you get a legendary proc and activate your 6sec Combustion and dont have Meteor available, that shouldnt count agianst you either.

Also the TCs are still going through Fire conduits and sims and legendaries and such, so its likely this may change once i get around to fire rotational stuff. But for now I just wanted the 6sec combusts to not skew their statistics.